packages/@webex/plugin-meetings/test/unit/spec/multistream/sendSlotManager.ts

```diff
@@ -0,0 +1,216 @@
+import SendSlotManager from '@webex/plugin-meetings/src/multistream/sendSlotManager';
+import { MediaType, MultistreamRoapMediaConnection } from "@webex/internal-media-core";
+import {expect} from '@webex/test-helper-chai';
+import sinon from 'sinon';
+
+describe('SendSlotsManager', () => {
+    let sendSlotsManager: SendSlotManager;
+    const LoggerProxy = {
+        logger: {
+            info: sinon.stub(),
+        },
+    };
+    
+    beforeEach(() => {
+        sendSlotsManager = new SendSlotManager(LoggerProxy);
+    });
+    
+    describe('createSlot', () => {
+        let mediaConnection;
+        const mediaType = MediaType.AUDIO_MAIN;
+
+        beforeEach(() => {
+            mediaConnection = {
+                createSendSlot: sinon.stub(),
+            } as MultistreamRoapMediaConnection;
+        });
+
+        it('should create a slot for the given mediaType', () => {
+            sendSlotsManager.createSlot(mediaConnection, mediaType);
+        
+            expect(mediaConnection.createSendSlot.calledWith(mediaType, true));
+        });
+
+        it('should create a slot for the given mediaType & active state', () => {
+            sendSlotsManager.createSlot(mediaConnection, mediaType, false);
+        
+            expect(mediaConnection.createSendSlot.calledWith(mediaType, false));
```

Comment:
do we not verify the number of calls?

Reply:
I don't think we need to because if the `sendSlotsManager.createSlot` is called with the same `mediaType` the second time, it will throw an error and in none of the meetings codes we call `mediaConnection.createSendSlot` directly.